hooks/pre/30-upgrade.rb

```diff
@@ -45,12 +45,12 @@ def remove_gutterball
 def migrate_pulp
   # Fix pid if neccessary
   if Kafo::Helpers.execute("grep -qe '7.[[:digit:]]' /etc/redhat-release")
-    Kafo::Helpers.execute("sed -i -e 's?/var/run/mongodb/mongodb.pid?/var/run/mongodb/mongod.pid?g' /etc/mongod.conf")
+    Kafo::Helpers.execute("sed -i -e 's?/var/opt/rh/rh-mongodb34/run/mongodb/mongod.pid?/var/opt/rh/rh-mongodb34/run/mongodb/mongod.pid?g' /etc/opt/rh/rh-mongodb34/mongod.conf")
   end
 
   # Start mongo if not running
```

Comment:
I wonder if we should just remove this block and the `pulp-manage-db`. If you remove `/var/lib/pulp/init.flag` then the installer will just migrate the DB again and ensure the service is running.

It may be needed to verify the `remove_node_importers` and `remove_node_distributors` can still run though.